framework/src/st_things/things_stack/src/common/framework/things_data_manager.c

```diff
@@ -1260,23 +1263,68 @@ static int parse_things_info_json(const char *filename)
 				cJSON *certificate = cJSON_GetObjectItem(file_path, KEY_CONFIGURATION_FILEPATH_CERTIFICATE);
 				cJSON *privateKey = cJSON_GetObjectItem(file_path, KEY_CONFIGURATION_FILEPATH_PRIVATEKEY);
 
-				if (NULL != svrdb && NULL != provisioning && NULL != certificate && NULL != privateKey) {
-					memset(g_svrdb_file_path, 0, (size_t) MAX_FILE_PATH_LENGTH);
-					memset(g_certificate_file_path, 0, (size_t) MAX_FILE_PATH_LENGTH);
-					memset(g_private_key_file_path, 0, (size_t) MAX_FILE_PATH_LENGTH);
-					memcpy(g_svrdb_file_path, svrdb->valuestring, strlen(svrdb->valuestring) + 1);
-					memcpy(g_certificate_file_path, certificate->valuestring, strlen(certificate->valuestring) + 1);
-					memcpy(g_private_key_file_path, privateKey->valuestring, strlen(privateKey->valuestring) + 1);
-					memcpy(g_things_cloud_file_path, provisioning->valuestring, strlen(provisioning->valuestring) + 1);
+				if (NULL == svrdb || NULL == provisioning || NULL == certificate || NULL == privateKey) {
+					THINGS_LOG_V_ERROR(THINGS_ERROR, TAG, "User certificate file not found");
+					return 0;
+				}
 
-					THINGS_LOG_D(THINGS_INFO, TAG, "Security SVR DB file path : %s", g_svrdb_file_path);
-					THINGS_LOG_D(THINGS_INFO, TAG, "[configuration] svrdb : %s / provisioning : %s", svrdb->valuestring, provisioning->valuestring);
-					THINGS_LOG_D(THINGS_INFO, TAG, "[configuration] certificate : %s / privateKey : %s", certificate->valuestring, privateKey->valuestring);
+				memset(g_svrdb_file_path, 0, (size_t) MAX_FILE_PATH_LENGTH + 1);
```

Comment:
You didn't change this. Please check my previous comment.

Reply:
modify